TOC.md

```diff
@@ -140,9 +140,8 @@
     - [TiSpark Topology](/tispark-deployment-topology.md)
     - [Cross-DC Topology](/geo-distributed-deployment-topology.md)
     - [Hybrid Topology](/hybrid-deployment-topology.md)
-  - Install and Start
-    - [Use TiUP](/production-deployment-using-tiup.md)
-    - [Deploy on Kubernetes](/tidb-in-kubernetes.md)
+  - [Deploy Using TiUP](/production-deployment-using-tiup.md)
+  - [Deploy on Kubernetes](/tidb-in-kubernetes.md)
```

Comment:
URL: "deploy-on-kubernetes.md"

Reply:
Generally, do not change file names unless we have special reasons or the file name completely deviates from the content.